.travis.yml

```diff
@@ -19,11 +19,10 @@ env:
 
 matrix:
   include:
-    - env: PLATFORM="i686"
     - env: PLATFORM="x86_64"
 
 script:
-  - PLATFORM=$PLATFORM TRAVIS_COMMIT=$TRAVIS_COMMIT ./build.sh
```

Comment:
Is using `travis_wait` unavoidable? It appears to hide all output until the command completes - and if the build hits the time limit (e.g. [this one](https://travis-ci.org/pypa/manylinux/builds/514342665)), you never see any output at all, so you can't see how far it got or if something was hanging.

Reply:
I'm trying something found there: https://github.com/travis-ci/travis-ci/issues/4190

Reply:
Ok, we have logs but build still times out.
The build is passing on my end... https://travis-ci.com/mayeut/manylinux/builds/107284605

Reply:
Now a 2 build stage with cache, logs are here and build build time can be over 50 minutes.